inc/Custom_Header/Component.php

```diff
@@ -69,27 +69,11 @@ public function wp_head_callback() {
 			return;
 		}
 
-		?>
-		<style type="text/css">
-		<?php
-		// Has the text been hidden?
-		if ( ! display_header_text() ) :
-			?>
-			.site-title,
-			.site-description {
-				position: absolute;
-				clip: rect(1px, 1px, 1px, 1px);
-			}
-			<?php
-			// If the user has set a custom color for the text use that.
-		else :
-			?>
-			.site-title a,
-			.site-description {
-				color: #<?php echo esc_attr( $header_text_color ); ?>;
-			}
-		<?php endif; ?>
-		</style>
-		<?php
+		if ( ! display_header_text() ) {
```

Comment:
Nice cleanup. On a side note, I hate this approach to hiding content. It goes against web standards best practices and feels really clunky. Inlining CSS via PHP to hide content is crude. If we can do the same by just toggling a class name, I'd be much happier.